src/Elmish.WPF/BindingData.fs

```diff
@@ -441,10 +527,10 @@ module internal BindingData =
 
 
   module TwoWay =
-  
-    let id<'a> : BindingData<'a, 'a> =
-      { TwoWayData.Get = box
-        Set = fun obj _ -> obj |> unbox }
+
+    let id<'a> : BindingData<'a, 'a, 'a> =
```

Comment:
The explicit type parameter on `id` was removed in the first commit and then added back in some subsequent commit.

Can it simply remain there in all the commits?

Reply:
Specifically, it's added back in the next commit: ee1b6a2a3c827cff4896dff23b9885ffc5404cbc

Reply:
Specifically, it's added back in the next commit: ee1b6a2a3c827cff4896dff23b9885ffc5404cbc

Reply:
@TysonMN I extracted the boxing change into its own PR #497